engines/drascula/drascula.cpp

```diff
@@ -928,9 +928,9 @@ bool DrasculaEngine::loadDrasculaDat() {
 	in.open(filename.c_str());
 
 	if (!in.isOpen()) {
-		Common::String errorMessage = Common::String::format(_("Unable to locate the '%s' engine data file."), filename.c_str());
+		Common::U32String errorMessage = Common::U32String::format(_("Unable to locate the '%s' engine data file."), filename.c_str());
 		GUIErrorMessage(errorMessage);
-		warning("%s", errorMessage.c_str());
+		warning("Unable to locate the '%s' engine data file.", filename.c_str());
 
 		return false;
 	}
```

Comment:
Should I do something like below to avoid copy paste? (here and some other files)
```suggestion
		if (!in.isOpen()) {
		Common::String msg = _s("Unable to locate the '%s' engine data file.");

		Common::U32String errorMessage = Common::U32String::format(_(msg), filename.c_str());
		GUIErrorMessage(errorMessage);
		warning(msg.c_str(), filename.c_str());

		return false;
	}
```

Reply:
I saw this yesterday, and I was wondering if it should be changed. The current code does not shock me, but I think your proposed change is a good idea.